play/src/components/ButtonSubmitAnswer.tsx

```diff
@@ -47,23 +52,25 @@ export default function ButtonSubmitAnswer({
       {isHint ? hintButtonText : buttonText}
     </Typography>
   );
+
   return isSelected && !isSubmitted ? (
     <GamePlayButtonStyled
       data-testid="answer-button-enabled"
       onClick={() => {
-        const answerText = selectedAnswer;
-        const answer = new LocalAnswer({
-          answerContent: {
-            rawAnswer: answerText ?? '',
-            normAnswer: [],
-          },
+        const submitAnswer = new BackendAnswer(
```

Comment:
update the creation of an answer to use `BackendAnswer` and compose in a `Answer` via the `AnswerFactory`